### PR TITLE
Fix MSAA performance on Python 3.11

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
https://github.com/nvaccess/nvda/pull/15544#issuecomment-1744208482

### Summary of the issue:
MSAA performance is extremely poor (over a second of lag on a high-end system).

### Description of how this pull request fixes the issue:
Removes the patched comtypes `replace_my_import` as suggested in https://github.com/nvaccess/nvda/pull/15544#issuecomment-1744853055.

### Testing strategy:
Followed STR, can no longer reproduce MSAA performance degradation.

### Known issues with pull request:
None known

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] API is compatible with existing add-ons.
- [x] Security precautions taken.

### CODE REVIEW

File: source/monkeyPatches/comtypesMonkeyPatches.py [hunk 1]
```diff
@@ -187,19 +187,6 @@ def replace_check_version() -> None:
 	comtypes._check_version = _check_version
 
 
-def new_my_import(fullname):
-	import comtypes.client._generate
-	importlib.invalidate_caches()
```
Comment (lukaszgo1) [inline source/monkeyPatches/comtypesMonkeyPatches.py:192]: Please remove, now unused, import of `importlib` at the top of file.

**Comment by lukaszgo1** [comment]
While  removing this monkey patch alongside Python 3.11 upgrade is obviously fine, I'm wondering if it wouldn't be better to submit PR removing it against master. The performance issues it causes are only under  3.11, but as I've explained in https://github.com/nvaccess/nvda/pull/15544#issuecomment-1744208482 the monkey patch is unnecessary since several versions of comtypes. Removing it on master would ensure Alpha users can test without it for longer period of time - I assume Python 3.11 upgrade won't be merged for a while yet.

**Comment by codeofdusk** [comment]
@lukaszgo1 Rebased on master.

**Review by seanbudd — APPROVED** [review]
Thanks @codeofdusk and @lukaszgo1

**Pull request merged**